srcs/Webserv/Config.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef SRCS_WEBSERV_CONFIG_HPP_
+#define SRCS_WEBSERV_CONFIG_HPP_
+
+#include <string>
+
+class Config {
+ public:
+  // TODO(iyamada) 取りあえず
+  Config() {}
+  ~Config() {}
+  Config(const Config& other) { *this = other; }
+  Config& operator=(const Config& other) {
+    if (this == &other) return *this;
+
+    this->port_ = other.port_;
+    return *this;
+  }
+  explicit Config(const std::string& port) : port_(port) {}
```

Comment:
Google Style Guideに則った順番で記述したいです。
以下の順番になります。コンストラクタとassignment opratorsは並列ですが、コンストラクタ->assignment operatorsの順番にしたいです。

- Constructors and assignment operators
- Destructor

```suggestion
  Config() {}
  explicit Config(const std::string& port) : port_(port) {}
  Config(const Config& other) { *this = other; }
 
  Config& operator=(const Config& other) {
    if (this == &other) return *this;

    this->port_ = other.port_;
    return *this;
  }
  
   ~Config() {}

```